deps/openssl/openssl/.gitattributes

```diff
@@ -0,0 +1,12 @@
+*.der binary
```

Comment:
this shouldn't be there either. Delete all of the dot files (e.g. `openssl/.gitattributes`, `openssl/.travis*`, etc)

Reply:
should i push another commit on top of these changes?

Reply:
Nevermind, I rebased the original commit and removed all the dot files for git and travis in the openssl/openssl directory. I had already removed the .git folder+files in the second commit because the build fails otherwise.